packages/web/components/Topbar/SearchBox.tsx

```diff
@@ -160,12 +160,12 @@ const SearchBox = () => {
           `
         )}
       >
-        <SearchIcon />
+        <SearchIcon isSearching={false} />
         <input
           ref={inputRef}
-          placeholder={t`search.search`}
+          placeholder={t`search.search`.toString()}
```

Comment:
Is this change necessary?

Reply:
It's detected by vscode, so I think it's necessary to fix that.